### ISSUE TITLE
Verified ITN Pool Icon not visible

### ISSUE THREAD

**Issue by AdabaseOne** [PegasusPool/pegasus-tool-features#12]
**Bug description**
My pool BASE1 is live since August 30th, and then on September 4th, I extended my metadata with the ITN signature. However on the Pegasus App, the green icon isn't visible, whereas on pooltool it is.

**Steps to reproduce the behavior**
1. Go to https://pooltool.io/
2. Search for BASE1
3. Take note of the green 'Verified ITN Pool' icon
4. Open Pegasus App
5. Search for BASE1
Problem: Green icon is not visible

**Expected behavior**
BASE1 should be marked as a verified ITN pool in the Pegasus App.

**Smartphone (please complete the following information):**
 - Device: Samsung S20
 - OS: Android
 - Browser: Chrome

**Additional context**
I was curious if there were any other pools facing this problem and after checking ~ 10 pools I found another one with the exact same behavior (not sure if I should mention the pool though..). I could imagine the problem occurs if the metadata extension is not registered with the initial pool registration, but at a later date.

**Pool information**
Metadata: https://www.adabase.one/pool-metadata.json
Extended metadata: https://www.adabase.one/extended-metadata.json
Pool ID: ca7045698268675561f5513e49ecd4d1f6fc413fcd08cb73778b2869
Pool Ticker: BASE1

**Comment by PegasusPool** [comment]
Hi,

Thanks for reaching out.
In the past I was contacted by some operators that the green shield is visible on pooltool but not in the pegasus app. Most of the time when I investigated it turned out that the extended metadata format was incorrect (was using an early version before the format was finalized) and that caused the reason for not showing the pool as verified in pegasus.

I have looked into your extended metadata and it's correct. As a result I fixed your pool's verified status in the Pegasus app.
